charts/harvester-csi-driver/Chart.yaml

```diff
@@ -18,12 +18,12 @@ keywords:
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.22
+version: 0.1.23
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
-appVersion: "v0.2.2"
+appVersion: "v0.2.3"
```

Comment:
The appVersion is associated with the Harvester CSI image version.
As you saw here (https://github.com/harvester/harvester-csi-driver/releases), the latest image version is `v0.2.2`.
We could skip updating the appVersion at this moment.

Just updating the Chart version from `0.1.22` -> `0.1.23` is enough.

Reply:
Fixed in 9b4f32733d4fde97b4cc2b003797304cea24f363.